src/main/resources/l10n/JabRef_es.properties

```diff
@@ -70,9 +70,6 @@ Attempting_SQL_export...=Intentando_la_exportación_SQL...
 Auto=Auto
 Autodetect_format=Autodetectar_formato
 Autogenerate_BibTeX_keys=Autogenerar_claves_BibTeX
-Autogenerate_BibTeX_keys=Autogenerar_claves_BibTex
-Autolink_files_with_names_starting_with_the_BibTeX_key=Autoenlazar_con_nombres_comenzando_por_la_clave_BibTeX
```

Comment:
Ahh, you also picked the wrong ones... ;-) I merged duplicate-free versions for sv, da, and es quite recently.